code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -193,14 +193,14 @@
 	
 	var/spot = src.loc
 	moveToNullspace()
-	if(spot == user)
+	if(spot == user && !user.get_inactive_held_item())
 		user.put_in_inactive_hand(prosthetic)
 	else
-		prosthetic.loc = spot
+		prosthetic.loc = get_turf(user)
```

Comment:
i guess technically you should be using forcemove here?

Reply:
sounds good, know of any better way to do the location thing though? i'd rather being able to do it in pockets/backpack but it gets all wonky. This change just makes it drop at the floor rather than actually let it work